sway-lib-std/src/low_level_call.sw

```diff
@@ -0,0 +1,105 @@
+library low_level_call;
+
+use ::assert::assert;
+use ::bytes::Bytes;
+use ::contract_id::ContractId;
+use ::option::Option;
+use ::revert::require;
+use ::vec::Vec;
+
+pub struct CallParams {
+    coins: u64,
+    asset_id: ContractId,
+    gas: u64,
+}
+
+// TODO : Replace with `from` when implemented
+fn contract_id_to_bytes(contract_id: ContractId) -> Bytes {
+    let mut target_bytes = Bytes::with_capacity(32);
+    target_bytes.len = 32;
+
+    __addr_of(contract_id).copy_bytes_to(target_bytes.buf.ptr, 32);
+
+    target_bytes
+}
+
+/// Represent a raw pointer as a Bytes
+fn ptr_as_bytes(ptr: raw_ptr) -> Bytes {
+
+    let mut bytes = Bytes::with_capacity(8);
+    bytes.len = 8;
+
+    // Need to copy pointer to heap so it has an address and can be copied onto the bytes buffer
+    let mut ptr_on_heap = Vec::new();
+    ptr_on_heap.push(ptr);
+    ptr_on_heap.buf.ptr.copy_bytes_to(bytes.buf.ptr, 8);
+
+    bytes
+}
+
+/// Call a target contract with an already-encoded payload
```

Comment:
Could you possibly add a simple 1-liner comment with a common use-case  for each of these functions to help make it super clear when you would choose one option over another?

Reply:
Currently this function is private, and only called by  `call_with_function_selector` after `create_payload` has been used to assemble the payload. In a future iteration (SDK support permitting) this function can be `pub` and called using a raw payload directly generated by the SDK.